### DISCUSSION TITLE
Does xournal++ support having multiple windows of different files open at once?

### DISCUSSION THREAD

**Discussion by sbalatbat** [xournalpp/xournalpp#4342] [General]
Hello, I was wondering if I can have multiple files open at once in Xournal++ cos I'm using it for college stuff and sometimes I need multiple open at once. If not, is there a way to implement it, and if yes, how do I utilize it?

**Comment by rolandlo** [comment]
You can only have multiple files open at once in Xournal++ if you run multiple instances of Xournal++.
